lib/MusicBrainz/Server/Entity/Recording.pm

```diff
@@ -63,7 +63,7 @@ around TO_JSON => sub {
 
     return {
         %{ $self->$orig },
-        isrcs   => [map { $_->isrc } $self->all_isrcs],
+        isrcs   => [map { $_->TO_JSON } @{$self->isrcs}],
```

Comment:
Weird, I guess the Flow type was wrong all this time. But if you change this, you'll need to fix the usages of the `isrcs` property in root/static/scripts/common/MB/Control/Autocomplete.js and root/static/scripts/common/entity.js which expect a list of strings.